code/game/objects/machinery/mortar.dm

```diff
@@ -258,12 +275,56 @@
 		return
 	return ..()
 
+// The big boy, the Howtizer.
+
+/obj/item/mortar_kit/howitzer
+	name = "\improper TA-100Y howitzer"
+	desc = "A manual, crew-operated and towable howitzer, will rain down 150mm laserguided and accurate shells on any of your foes. Right click to anchor to the ground."
+	icon = 'icons/Marine/howitzer.dmi'
+	icon_state = "howitzer"
+	max_integrity = 400
+	flags_item = IS_DEPLOYABLE|TWOHANDED|DEPLOYED_NO_PICKUP|DEPLOY_ON_INITIALIZE
+	w_class = WEIGHT_CLASS_HUGE
+	deployed_item = /obj/machinery/deployable/mortar/howitzer
+
+/obj/machinery/deployable/mortar/howitzer
+	anchored = FALSE // You can move this.
+	offset_per_turfs = 25 // Howizters are significantly more accurate.
+	travel_time = 60
+	fire_sound = 'sound/weapons/guns/fire/howitzer_fire.ogg'
+	reload_sound = 'sound/weapons/guns/interact/tat36_reload.ogg'
+	fall_sound = 'sound/weapons/guns/misc/howitzer_whistle.ogg'
+	allowed_shells = list(
+		/obj/item/mortal_shell/howitzer,
+		/obj/item/mortal_shell/howitzer/white_phos,
+		/obj/item/mortal_shell/howitzer/he,
+		/obj/item/mortal_shell/howitzer/incendiary,
+		/obj/item/mortal_shell/howitzer/plasmaloss,
+		/obj/item/mortal_shell/flare,
+	)
+
+/obj/machinery/deployable/mortar/howitzer/attack_hand_alternate(mob/living/user)
+	if(!Adjacent(user) || user.lying_angle || user.incapacitated() || !ishuman(user))
+		return
+
+	if(!anchored)
+		anchored = TRUE
+		to_chat(user, span_warning("You have anchored the gun to the ground. It may not be moved."))
+	else
+		anchored = FALSE
+		to_chat(user, span_warning("You unanchored the gun from the gruond. It may be moved."))
```

Comment:
early return
this doesnt even need to be an if use 
ancored = !anchored
and ? :